app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,249 @@
+    <string name="menu_item_title_delete_favorite">Usuń</string>
+    <string name="menu_item_title_set_alarm">Ustaw przypomnienie</string>
+    <string name="menu_item_title_delete_alarm">Wyczyść przypomnienie</string>
+    <string name="menu_item_title_delete_all">Usuń wszystkie</string>
+    <string name="menu_item_title_favorites">Ulubione</string>
+    <string name="menu_item_title_feedback">Informacja zwrotna</string>
+    <string name="menu_item_title_navigate">Przejdź</string>
+    <string name="menu_item_title_refresh">Odśwież</string>
+    <string name="menu_item_title_share_session">Udostępnij wydarzenie</string>
+    <string name="menu_item_title_share_favorites">Udostępnij ulubione</string>
+    <string name="menu_item_title_share_session_text">jako tekst</string>
+    <string name="menu_item_title_share_session_json">jako JSON (dla Chaosflix)</string>
+    <string name="share_as_text">Udostępnij jako tekst</string>
+    <string name="share_as_json">Udostępnij jako JSON (dla Chaosflix)</string>
+
+    <!-- AlertDialog -->
+    <string name="OK">OK</string>
+
+    <!-- CertificateErrorFragment -->
+    <string name="certificate_error_message">"
+        Nie udało się nawiązać bezpiecznego połączenia.\n
+        (<xliff:g example="Unknown Error" id="error">%s</xliff:g>)
+    </string>
+    <string name="certificate_error_title">Nie udało się dodanie certyfikatu.</string>
+
+    <!-- Error messages -->
+    <string name="dlg_err_connection_failed">Problem z połączeniem</string>
+    <string name="dlg_err_failed_unknown_host">Nieznany adres:\n<xliff:g example="http://example.com" id="url">%s</xliff:g></string>
+    <string name="dlg_err_failed_wrong_http_credentials">Błędna autentykacja HTTP</string>
+    <string name="dlg_err_failed_timeout">Serwer nie odpowiada (timeout).</string>
+    <string name="dlg_err_failed_connect_failure">Nieznany błąd połączenia</string>
+    <string name="dlg_err_failed_not_found">Błędny URL grafiku</string>
+    <string name="dlg_err_failed_parse_failure">Nie mogłem przetworzyć odpowiedzi</string>
+    <string name="dlg_err_failed_ssl_failure">błąd SSL</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HTTP niedozwolone. Proszę użyć HTTPS.</string>
+    <string name="dlg_err_schedule_data">Błąd danych grafiku</string>
+    <string name="dlg_err_schedule_data_empty">
+        Nic do wyświetlenia. Grafik w wersji \"<xliff:g example="0.99" id="version">%s</xliff:g>\"
+        nie zawiera żadnych sesji.
+    </string>
+    <string name="dlg_err_schedule_data_empty_without_version">"
+        Nic do wyświetlenia. Grafik nie zawiera żadnych sesji.
+    </string>
+    <string name="uptodate">Grafik jest aktualny.</string>
+
+    <!-- Schedule changes dialog -->
+    <string name="schedule_changes_dialog_title">Aktualizacja grafiku</string>
+    <string name="schedule_changes_dialog_updated_to_text">Zaktualizowano grafik do wersji\\u0020</string>
```

Comment:
There is one backslash to much. I will change `\\u0020` to `\u0020`.

Reply:
Is this the only place where its used? It is not in nominativ (and I need to change it), I thought it is.

Reply:
I don't understand to which word you are referring as nominativ here. Did you comment on the right place?

Reply:
> Here is an example screenshot of where the texts you translated are used:
> 
> ![screenshot](https://user-images.githubusercontent.com/144518/145975372-27745c0d-d2d9-43c0-80ed-8b2744faaae6.png)

I was referring to the nominativ regarding the screenshot. Fixing itnow , since I thought that this was in nominativ, but it is not. And I'm looking at other changes introduced earlier today. In one hour I should be ready.